jdaviz/configs/default/plugins/model_fitting/model_fitting.py

```diff
@@ -528,7 +528,7 @@ def _check_model_component_compat(self, axes=['x', 'y'], display_units=None):
         self._check_model_equation_invalid()
 
     def _on_global_display_unit_changed(self, msg):
-        axis = {'spectral': 'x', 'flux': 'y'}.get(msg.axis)
+        axis = {'spectral': 'x', 'spectral_y': 'y'}.get(msg.axis)
```

Comment:
this probably needs some logic to skip if the `msg.axis` is not either `spectral` or `spectral_y` (could be by checking if `axis` is `None` after the get, or rewriting this into `if, elif, else: return`.

Reply:
done, can i resolve?